_posts/2018-04-17-devx-at-iw.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: DevEx at Infinity Works
+author: paul_lucas
+image: laptop_desk.jpg
+---
+
+We attended [QCon in London](https://qconlondon.com) last month (which as an aside, I highly recommend if you've not been before) and one of the day's tracks that caught my attention was looking at "DevEx" or "Developer Experience" - the daily life of developers/engineers and how that might be improved through better tooling, process and skills.  It got me thinking - what is the "DevEx" for engineers at Infinity Works and what are the things we can do to nuture it?
+
+> "DevEx is not just about the tools, but the daily experience at work"
+> - QCon Presentation
+
+As a services business it's often difficult to convey to people what it means to join us since we work on such a diverse set of things - we don't have a product we can point at or a particular technical stack that we all use, in fact the answer to a lot of questions that people ask is "it depends"!
+
+So what _does_ it look like to be part of Infinity Works?  What is our day-to-day "DevEx"?
+
+## "There's _always_ something going on, Danny!"
+
+Engineering at Infinity Works is first and foremost about solving real-world problems.  We work as trusted delivery partners for our clients helping them to deliver their projects, figuring out solutions to hard technical problems and bringing a strong culture of Agile and DevOps along with us.
+
+The projects and challenges that we tackle are many and varied and that variety is really at the heart of our DevEx.
+
+We push ourselves to learn different domains, experience different ways of working and understand a wide range of technical stacks.  We work across the stack and across the SDLC.  Engineering to us doesn't just mean being able to be a great developer, it also means helping out with the infrastructure-as-code, building the deployment pipelines and integrating automated operational monitoring.
```

Comment:
Is it worth linking out to SDLC on wiki, incase people don't immediately understand the initialism?

Reply:
Changed wording